client/src/entrypoints/admin/core.js

```diff
@@ -150,27 +152,26 @@ function enableDirtyFormCheck(formSelector, options) {
       $form.on('change keyup', updateDirtyCheck).trigger('change');
 
       const validInputNodeInList = (nodeList) => {
-        for (const node of nodeList) {
+        nodeList.forEach((node) => /* eslint-disable-line consistent-return */ {
```

Comment:
Be careful here with this change. 

The for loop return statement on line 160 below is actually returning from this function - when you use the forEach the rest of the results still go through the mapping. 

You either need to set a variable above the loop like `let isValidNode = false;` and then set the variable to true when the condition matches. 
Alternatively you can use array.some 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some